buddyboss/buddyboss-platform
# 880 - Fixed URL preview generation

### All Submissions:

Post this in the activity feed;
2020년 5월 6일 1번째 교실

Learnt

– 지난주에 뭐 했어요? Q & A [Break the fear in speaking Korean]

– 도 usage (same subject and different subject and positive and negative)

– Contracted form

– 거 vs 꺼

– 한국어 문장의 형태 (preposition vs postposition)

– 은/는/이/가 https://youtu.be/fCxLNRLntc0?t=53

숙제

– Short Contracted Form 숙제

다음주

– 지난주에 뭐 했어요

– 도 concept review

– Workbook

– 은/는/이/가

– 9W6H

Numbers






### Changes proposed in this Pull Request:



Fixes #880 .

### How to test the changes in this Pull Request:

1. Go to the activity feed
2. Post the content shared in the details
3. Post the same content in English
4. See error

### Proof Screenshots or Video
Non-EN: http://prntscr.com/tf4isq
EN: http://prntscr.com/tf4iy0
URL Prev: http://prntscr.com/tf4j73


### Other information:




### Changelog entry

> Fixed URL preview generation
